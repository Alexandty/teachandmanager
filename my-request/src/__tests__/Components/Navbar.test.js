import React from 'react';
import { shallow } from 'enzyme';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import NavbarApp from './../../Components/Navbar';
configure({ adapter: new Adapter() });

describe('Test navbar', () => {
    it('renderizar navbar ', () => {
        const wrap = shallow(<NavbarApp />);
        expect(toJson(wrap)).toMatchSnapshot();
    });
});