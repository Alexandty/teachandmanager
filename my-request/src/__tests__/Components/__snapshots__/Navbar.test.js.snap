// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Validando navbar toMatchSnapshot renderizar navbar  1`] = `
<Uncontrolled(Navbar)
  bsStyle="default"
  collapseOnSelect={true}
  inverse={true}
>
  <NavbarHeader>
    <NavbarBrand>
      <img
        alt="logo"
        height="80"
        src="http://www.techandsolve.com/wp-content/themes/techandsolve/images/logo-tech-and-solve-footer.png"
        width="100"
      />
    </NavbarBrand>
    <NavbarToggle />
  </NavbarHeader>
  <NavbarCollapse>
    <Nav
      bsClass="nav"
      justified={false}
      pullLeft={false}
      pullRight={false}
      stacked={false}
    >
      <NavItem
        active={false}
        disabled={false}
        eventKey={1}
      >
        <Link
          replace={false}
          to="/"
        >
          Inicio
        </Link>
      </NavItem>
      <NavDropdown
        eventKey={3}
        id="basic-nav-dropdown"
        title="Vacaciones"
      >
        <MenuItem
          bsClass="dropdown"
          disabled={false}
          divider={false}
          eventKey={3.1}
          header={false}
        >
          <Link
            replace={false}
            to="/consulta"
          >
            Consulta de solicitudes
          </Link>
        </MenuItem>
        <MenuItem
          bsClass="dropdown"
          disabled={false}
          divider={false}
          eventKey={3.2}
          header={false}
        >
          <Link
            replace={false}
            to="/solicitud"
          >
            Solicitud Vacaciones
          </Link>
        </MenuItem>
        <MenuItem
          bsClass="dropdown"
          disabled={false}
          divider={false}
          eventKey={3.3}
          header={false}
        >
          <Link
            replace={false}
            to="/solicitudeslider"
          >
            Mis Solicitudes
          </Link>
        </MenuItem>
      </NavDropdown>
    </Nav>
    <Nav
      bsClass="nav"
      justified={false}
      pullLeft={false}
      pullRight={true}
      stacked={false}
    />
  </NavbarCollapse>
</Uncontrolled(Navbar)>
`;
